btcpay-setup.sh

```diff
@@ -327,50 +388,54 @@ if [ "$BTCPAYGEN_OLD_PREGEN" == "true" ]; then
 fi
 
 # Schedule for reboot
-if [ -x "$(command -v systemctl)" ]; then # Use systemd
-if [ -e "/etc/init/start_containers.conf" ]; then
-    echo -e "Uninstalling upstart script /etc/init/start_containers.conf"
-    rm "/etc/init/start_containers.conf"
-    initctl reload-configuration
-fi
-echo "Adding btcpayserver.service to systemd"
-echo "
-[Unit]
-Description=BTCPayServer service
-After=docker.service network-online.target
-Requires=docker.service network-online.target
-
-[Service]
-Type=oneshot
-RemainAfterExit=yes
-
-ExecStart=/bin/bash -c  '. /etc/profile.d/btcpay-env.sh && cd \"\$BTCPAY_BASE_DIRECTORY/btcpayserver-docker\" && . helpers.sh && btcpay_up'
-ExecStop=/bin/bash -c   '. /etc/profile.d/btcpay-env.sh && cd \"\$BTCPAY_BASE_DIRECTORY/btcpayserver-docker\" && . helpers.sh && btcpay_down'
-ExecReload=/bin/bash -c '. /etc/profile.d/btcpay-env.sh && cd \"\$BTCPAY_BASE_DIRECTORY/btcpayserver-docker\" && . helpers.sh && btcpay_restart'
-
-[Install]
-WantedBy=multi-user.target" > /etc/systemd/system/btcpayserver.service
-
-if ! [ -f "/etc/docker/daemon.json" ]; then
-echo "{
-\"log-driver\": \"json-file\",
-\"log-opts\": {\"max-size\": \"5m\", \"max-file\": \"3\"}
-}" > /etc/docker/daemon.json
-echo "Setting limited log files in /etc/docker/daemon.json"
-systemctl restart docker
+if [[ -x "$(command -v systemctl)" ]]; then
+	# Use systemd
+	if [[ -e "/etc/init/start_containers.conf" ]]; then
+		echo -e "Uninstalling upstart script /etc/init/start_containers.conf"
+		rm "/etc/init/start_containers.conf"
+		initctl reload-configuration
+	fi
+	echo "Adding btcpayserver.service to systemd"
+	echo "
+	[Unit]
```

Comment:
don't change the indentation here, this will change indentation in the systemctl.

Reply:
You're right. Fixed it!